requirements-ocr.txt

```diff
@@ -0,0 +1,12 @@
+pygments~=2.4.2
```

Comment:
A lot of these requirements are duplicated from the top-level requirements.txt.

If this example has extra reqs, just place them in the examples/ directory (sibling to the code).

Reply:
Thanks! That makes sense. :)